public/homepage/scripts/main.js

```diff
@@ -1,13 +1,3 @@
-/* global requirejs */
```

Comment:
This comment can't be removed, it's an instruction to our linters about a global variable.